pkg/artifactory/resource_artifactory_local_npm_repository.go

```diff
@@ -0,0 +1,34 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var npmLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{})
+
+func resourceArtifactoryLocalNpmRepository() *schema.Resource {
+
+	return mkResourceSchema(npmLocalSchema, universalPack, unPackLocalNpmRepository, func() interface{} {
+		return &NpmLocalRepositoryParams{
+			LocalRepositoryBaseParams: LocalRepositoryBaseParams{
+				PackageType: "npm",
+				Rclass:      "local",
+			},
+		}
+	})
+}
+
+type NpmLocalRepositoryParams struct {
+	LocalRepositoryBaseParams
+	Key string `hcl:"key" json:"key"`
```

Comment:
The key already exists in the `LocalRepositoryBaseParams. please look at `services.NpmLocalRepositoryParams`
Delete the key

Reply:
Deleted the key.